Makefile

```diff
@@ -37,7 +37,7 @@ GOPATH ?= $(shell cd ${ISTIO_GO}/../../..; pwd)
 export GOPATH
 
 # If GOPATH is made up of several paths, use the first one for our targets in this Makefile
-GO_TOP := $(shell echo ${GOPATH} | cut -d ':' -f1)
+GO_TOP := $(shell realpath ${GOPATH} | cut -d ':' -f1)
```

Comment:
`realpath` isn't in OSX.  This change would add a dev setup dependency.

Reply:
realpath is used here
https://github.com/istio/istio/blob/048c291ef7b233be3cc01fc36b0435bb2f420d2a/Makefile#L18 

and seems be a common way in Makefile.

Reply:
d'oh yeah, looks like that's a GNU makefile function.  TIL :-)   Sorry for the noise.